handler.go

```diff
@@ -20,7 +20,7 @@ type MountHandler func(ctx context.Context, c Socket) (interface{}, error)
 
 // RenderHandler the func that is called to render the current state of the
 // data for the socket.
-type RenderHandler func(ctx context.Context, data interface{}) (io.Reader, error)
+type RenderHandler func(ctx context.Context, rc *RenderContext) (io.Reader, error)
```

Comment:
This breaks live-contrib:

https://github.com/jfyne/live-contrib/blob/main/livefiber/template.go#L15